docs/examples.rst

```diff
@@ -1,17 +1,186 @@
+  Datasets for scalable and reproducible input pipelines. This is the original
+  codebase of ViT, MLP-Mixer, LiT, UViM, and many more models.
+
+- `T5X <https://github.com/google-research/t5x>`__  is a modular, composable,
+  research-friendly framework for high-performance, configurable, self-service
+  training, evaluation, and inference of sequence models (starting with
+  language) at many scales.
+
+Community Examples
+###################
+
+In addition to the curated list of official Flax examples, there is a growing
+community of people using Flax to build new types of machine learning models. We
+are happy to showcase any example built by the community here! If you want to
+submit your own example, we suggest that you start by forking one of the
+official Flax example, and start from there.
+
+
+.. list-table::
+    :header-rows: 1
+
+    * - Link
+      - Author
+      - Task type
+      - Reference
+    * - `matthias-wright/flaxmodels <https://github.com/matthias-wright/flaxmodels>`__
+      - `@matthias-wright <https://github.com/matthias-wright>`__
+      - Various
+      - GPT-2, ResNet, StyleGAN-2, VGG, ...
+    * - `DarshanDeshpande/jax-models <https://github.com/DarshanDeshpande/jax-models>`__
+      - `@DarshanDeshpande <https://github.com/DarshanDeshpande>`__
+      - Various
+      - Segformer, Swin Transformer, ... also some stand-alone layers
+    * - `google/vision_transformer <https://github.com/google-research/vision_transformer>`__
+      - `@andsteing <https://github.com/andsteing>`__
+      - Image classification, image/text
+      - https://arxiv.org/abs/2010.11929, https://arxiv.org/abs/2105.01601, https://arxiv.org/abs/2111.07991, ...
+    * - `JAX-RL <https://github.com/henry-prior/jax-rl>`__
+      - `@henry-prior <https://github.com/henry-prior>`__
+      - Reinforcement learning
+      - N/A
+    * - `DCGAN <https://github.com/bkkaggle/jax-dcgan>`__
+      - `@bkkaggle <https://github.com/bkkaggle>`__
+      - Image Synthesis
+      - https://arxiv.org/abs/1511.06434
+    * - `BigBird Fine-tuning <https://github.com/huggingface/transformers/tree/master/examples/research_projects/jax-projects/big_bird>`__
+      - `@vasudevgupta7 <https://github.com/vasudevgupta7>`__
+      - Question-Answering
+      - https://arxiv.org/abs/2007.14062
+    * - `jax-resnet <https://github.com/n2cholas/jax-resnet>`__
+      - `@n2cholas <https://github.com/n2cholas>`__
+      - Various resnet implementations
+      - `torch.hub`
+
+Contributing Policy
+********************
+If you are interested in adding an project to the Community Examples section, take the following into consideration:
+
+* **Examples**: examples should contain a README that is helpful, clear, and makes it easy to run the code. The code itself should be easy to follow.
+* **Tutorials**:  tutorials must preferably be run-able notebooks, be well written, and discuss an interesting topic. 
```

Comment:
another consideration: tutorials should be complementary to already existing documentation